### PR TITLE
JSON support in dialog, misc

### PR DESCRIPTION
**Pull request by hza**
* added JSON option and support to the dialog.

Screenshot 2023-07-16 at 14 09 03

Screenshot 2023-07-16 at 14 09 10

* Added some background in logo.

* Changed untilBuild (may not help) because I have this error:

Screenshot 2023-07-16 at 14 07 29

* some refactoring.

### CODE REVIEW

**Comment by chocovon** [comment]
The compatibility is limited by manual setting in the plugin page, I just released them and will fix the warning of deprecated API later.

TL;DR

The compatibilty issues raised after a long time I finished developing, at that time I found it annoying so I simply set the untilBuild ~213 on plugin webpage and filtered the email warning since then.
But when I upload the new ver 2.0.0, I've noticed that the version I set ~213 build to is 1.4.0, left the 1.3.0 still available to later IDEA builds, so I added ~213 limit to both 1.3.0 and 2.0.0, without noticing total unvailabilty in IDEA 2022 or later:(
Now I've removed the untilBuild limit, it would be available to latest builds, though the warning message would still spam my email if I don't fix them.

**Comment by hza** [comment]
I am trying to cover remainingFieldsCode by jUnit but without success.

Screenshot 2023-07-17 at 09 57 41

if you remember the case please let me know.

**Comment by chocovon** [comment]
I forgot how exactly they works, but the example in the inner-tool/src/test/java/Tester.java may help.
I've just tried it with 3 versions, the old v1.4.0 generated desired result:
Tester tester3 = new Tester();
Tester tester2 = new Tester();
tester2.setName("t1");
tester2.setType("tt");
tester2.obj = null;
Tester tester = new Tester();
tester.setName("t3");
tester.setType("hh");
tester.obj = tester2;
tester3.setName("t2");
tester3.setType("pp");
tester3.obj = tester3;
HashMap hashMap = new HashMap();
hashMap.put(tester, tester);
hashMap.put(tester2, tester3);
hashMap.put(tester3, tester2);
hashMap

However in v2.0.0, the variable name somehow becomes "obj", which is still usable though; but in your latest commit, the result is wrong:
...
Tester tester = new Tester();
tester.setName("t2");
tester.setType("pp");
tester.obj = obj;  // wrong code
...

But again thank you for your contribution, and hopefully you could fix them right :)

**Comment by hza** [comment]
fixed in [6ae8249](https://github.com/chocovon/debug-variable-sl/pull/2/commits/6ae82498ee294e6b94c64e20cec2c6ba933b9c76), my wrong refactoring.
